utils/build-script-impl

```diff
@@ -2065,6 +2065,25 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_TOOLS_DEPLOYMENT_TARGETS[@]}"
             ${DISTCC_PUMP} "${CMAKE}" --build "${build_dir}" $(cmake_config_opt ${product}) -- ${BUILD_ARGS} ${build_targets[@]}
             { set +x; } 2>/dev/null
         fi
+
+        # When we are building LLVM create symlinks to the c++ headers.
+        if [[ "${product}" == "llvm" ]]; then
+
+          # Find the location of the c++ header dir.
+          if [[ "$(uname -s)" == "Darwin" ]] ; then
+            HOST_CXX_DIR=$(dirname ${HOST_CXX})
+            HOST_CXX_HEADERS_DIR="$HOST_CXX_DIR/../../usr/include/c++"
```

Comment:
I know that this code can be written in one line, but I intentionally split this into two lines and introduced a variable name to make the code more readable.